charts/astronomer/values.yaml

```diff
@@ -193,6 +193,17 @@ houston:
     # Default here is to run at midnight every night
     schedule: "0 0 * * *"
 
+  # Check for Astronomer Runtime Updates
+  # This runs as a CronJob
+  updateRuntimeCheck:
+    # Enable check updates CronJob
+    enabled: true
+
+    # url for update service
+    url: https://updates.astronomer.io/astronomer-runtime
+
+    # Default here is to run at midnight every night
```

Comment:
We should pick a different minute to run, in order to avoid coinciding with every other job on the same node that runs on the hour. This will spread system load out more, reducing resource contention. Even 3 after would be better. Or 27 after. Or 43 after.

Reply:
fixed.